src/utils.h

```diff
@@ -2,13 +2,18 @@
 #ifndef DUNST_UTILS_H
 #define DUNST_UTILS_H
 
+#include <stdlib.h>
```

Comment:
`size_t` is defined in `stddef.h`. While `stddlib` includes `stddef` it's usually best practice to use the more specific header file.